spec/models/article_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+	subject { Article.new(title: "Great concepts of Rails")} 
+	before { subject.save }
+	
+	it 'title should be present' do
```

Comment:
[Optional] you can also check your associations/validations in your model tests using methods from [this gem](https://github.com/thoughtbot/shoulda-matchers).